code/platformio_override.ini.example

```diff
@@ -0,0 +1,37 @@
+# -------------------------------------------------------------------------------------------------------------
+# Example file for 'platformio_override.ini' use.
+# Either copy and paste this file, then rename removing the .example or create your
+# own file: 'platformio_override.ini'
+# 'platformio_override.ini' allows users to create their own environments to add, remove or change
+# firmware configurations. 
+#
+# See: https://github.com/xoseperez/espurna/wiki/PlatformIO#customize-build-settings
+# for more details.
+# -------------------------------------------------------------------------------------------------------------
+
+// Example for a standard Sonoff Basic R2 or R3 but enabling DHT support on GPIO 2.
+// It can be built from VSCode/PlatformIO by selecting the 'my-sonoff-build-dht' 

```

Comment:
"selecting ... build selection" -> "selecting the 'Build: my-sonoff-build-dht'"?

Reply:
Again yes, that makes more sense.